systemd/cloud-init-network.service.tmpl

```diff
@@ -9,7 +9,9 @@ Wants=cloud-init-local.service
 Wants=sshd-keygen.service
 Wants=sshd.service
 After=cloud-init-local.service
+{% if variant not in ["debian", "ubuntu"] %}
```

Comment:
I don't think it is "safe to include debian in this conditional here. There are plenty of images, even cloud image containing cloud-init, which also don't contain netplan.io in the image: `lxc launch images:debian/bookworm/cloud`. In these cases, I'd not want default packaging to drop this ordering on Debian.